### DISCUSSION TITLE
构建自己的信息简报 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#126] [Announcements]
# 构建自己的信息简报 · 构建我的被动收入

使用RSS与Email打造自己的信息简报。

https://www.bmpi.dev/self/use-rss-email-read/

**Comment by standin000** [comment]
邮箱客户端有适合墨水屏幕的吗？rss客户端feedme适配很不错

**Comment by sunxyz** [comment]
有一个 叫webfollow.cc 的网站提供了一个 rss->newsletter 的功能

**Comment by EmmettHan** [comment]
博主的博客对我有非常多的启发，正是通过博主的“打造高效信息输入渠道”了解到了RSS，但是似乎博主的博客并没有提供RSS链接，即使我使用了RSSHub Radar也没有找到， 博主是否有相关的打算呢？

**Comment by madawei2699** [comment]
https://www.bmpi.dev/index.xml

EmmettHan ***@***.***>于2025年7月30日 周三22:27写道：
